docs/index.asciidoc

```diff
@@ -27,6 +27,21 @@ By default, each event is assumed to be one line. If you
 want to join lines, you'll want to use the multiline codec.
 
 
+[id="plugins-{type}s-{plugin}-ecs"]
+==== Compatibility with the Elastic Common Schema (ECS)
+
+This plugin adds extra fields about the event's source, and can be configured to do so
+in an {ecs-ref}[ECS-compatible] way with <<plugins-{type}s-{plugin}-ecs_compatibility>>.
```

Comment:
```suggestion
This plugin adds extra fields about the event's source. 
Configure the <<plugins-{type}s-{plugin}-ecs_compatibility>> option to ensure
that these fields are compatible with {ecs-ref}[ECS].


```

Reply:
The link text should give users a clear idea of where the link will take them. "ECS-compatible" is not 100% clear and looks a lot like the link for "ecs_compatibility" option.  My intention is to have the "ECS" text link to "ECS," and "ecs_compatibility" link to the "ecs_compatibility" option.